### PR TITLE
update oktaUsers with data from okta on login

### LINKED ISSUE #3089: [Dev] Store Okta User data on login
Feature -  
When a user logs in we should contact okta to get their data and store their data in the okta users table.  This table can be expanded to include the expected fields from Okta (displayName, email, secondEmail, primaryPhone, mobilePhone) or that data can be stored as a json string in the metadata field of the oktaUser table.  

### This task is done when...
 - [ ] a user has an up to date row in the oktaUsers table when they log in 
 - [ ] Any data that has changed since they last logged in has been updated (you can manually update the DB row to check this)
 - [ ] Using this data is out of scope and will be covered in additional tickets.

### PR DESCRIPTION
**Pull request by knollfear**
…  This will prevent some chattiness wit Okta.

resolves #3089

**Description-**

**This pull request changes...**

- expands the fields of the okta_users table (run docker-compose exec api npm run migrate to ensure DB is ready)
- creates or updates okta user data on login (technically when calling the /me endpoint)

**This pull request also touches…**

- possible side effects of this change:
- unlikely to be any.  login may be slightly slower

**This pull request was tested in the follow ways…**

- table okta_users should be written to during login process.

**Steps to manually verify this change...**

1. docker-compose exec api npm run migrate
2. establish a baseline by running select * from okta_users in the hitech_apd schema
3. manipulate rows as you see fit (delete row, alter data, etc)
3. log in with a user
4. verify that data has changed for that user
5. change (or delete) data again
6. use system as normal (open an APD, edit something, etc)
7. verify that data has not changed

### This pull request is ready to review when...

- [ ] Automated tests are updated (and all tests are passing)
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate
- [x] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [ ] Code has been reviewed by someone other than the original author
- [x] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Review by thetif — APPROVED** [review]
looks good, great work

File: api/db/oktaUsers.test.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+} = require('./oktaUsers')
+
+tap.test('database wrappers / oktaUsers', async oktaUsersTests => {
+  //
+  // Life is like a hurricane
+  // Here in Duckburg
+  // Race cars, lasers, aeroplanes
+  // It's a duck-blur!
+  // Might solve a mystery
+  // Or rewrite history!
```
[the remark this answers is not in the document]
Comment (tbolt) [reply api/db/oktaUsers.test.js:21]: I'm ashamed I had to look this up

**Review by tbolt — APPROVED** [review]

**Review by eStar386 — APPROVED** [review]
Confirmed functionality of development environment that the okta_users table is active upon login for new and existing users. 
https://docs.google.com/spreadsheets/d/1mW6m0mQ2cxV-KbjNwVHTdQ1RhhB0D43wZZGQrQDOGbI/edit?usp=sharing

**Review by beparticular — APPROVED** [review]
Spoke with Tif on how to test. I am able to successfully log in to multiple accounts on this branch with the keybase users. Let me know if I need to check anything else.

**Review by NAretakis — APPROVED** [review]
Good to go.

**Pull request merged**